internal/options/options.go

```diff
@@ -75,6 +80,7 @@ func (o *Options) AddFlags() {
 	o.flags.BoolVar(&o.Version, "version", false, "kube-events-exporter version information")
 
 	o.flags.StringArrayVar(&o.EventTypes, "event-types", []string{EventTypesAll}, "Event types filter. Defaults to all types.")
+	o.flags.StringArrayVar(&o.InvolvedObjectAPIGroups, "involved-object-api-groups", []string{APIGroupsAll}, "Events involved object API groups filter. Defaults to all API groups.")
```

Comment:
I see what you meant with the flags the other day. It can be confusing if you don't know what involved object is. Hmm lets leave it as is, but if folks ask questions we can change. WDYT?

Reply:
What does this filter do, exclude or include the APIs, it is not clear from the command line args to me.

Reply:
This flag is meant to include API groups. I updated the help message to "List of included Event involved object API groups.". I also updated the other filtering flags to make them clearer.